.gitignore

```diff
@@ -12,3 +12,5 @@ target/
 build-storybook.log
 logs/
 storybook-static/
+report.20210513.101609.23053.0.001.json
+report.20210517.154123.44099.0.001.json
```

Comment:
we can add the `package-lock.json` here.
also - any idea what these `report` json files are?

Reply:
I don't know where the report files came from! Shall I delete them?

Reply:
Yeah I think that makes sense 👍